pkg/extensions/search/common/common.go

```diff
@@ -75,6 +75,24 @@ func GetImageDirAndTag(imageName string) (string, string) {
 	return imageDir, imageTag
 }
 
+func GetImageLastUpdated(imageInfo ispec.Image) time.Time {
```

Comment:
Can we add a comment for this method 

e.g. 
`// GetImageLastUpdated This method will return last updated timestamp. If history field is provided it will return timestamp of last entry in history`.

Feel free to update the comment accordingly.